src/highlight/HighlightCanvas.tsx

```diff
@@ -5,11 +5,12 @@ import './HighlightCanvas.scss';
 
 export type Props = {
     activeId: string | null;
-    annotations: AnnotationHighlight[];
+    annotations: Pick<AnnotationHighlight, 'id' | 'target'>[];
 };
 
-export default class HighlightCanvas extends React.Component<Props> {
+class HighlightCanvas extends React.Component<Props> {
     static defaultProps = {
+        activeId: null,
```

Comment:
Is this `null` value treated differently from undefined? If not, can we avoid using it?